VideoLocker/res/values/strings.xml

```diff
@@ -477,6 +477,14 @@
     <!-- In Sign up screen -->
     <string name="or_sign_up_with_email">or sign up with email</string>
 
+    <!-- New Logistration Launch screen text -->
+    <!-- Button leading user to create account screen -->
+    <string name="launch_register">Register</string>
+    <!-- Button leading user to Find Courses screen from the launch screen -->
+    <string name="launch_discover_courses">@string/find_courses_title</string>
```

Comment:
@BenjiLee What's the reason for the change? I might have missed some discussion thread, sorry.